src/Blocks/Button.purs

```diff
@@ -5,7 +5,7 @@ import Prelude
 import DOM.HTML.Indexed (HTMLbutton, HTMLdiv)
 import Halogen.HTML as HH
 import Halogen.HTML.Properties as HP
-import Ocelot.Core.Utils ((<&>))
+import Ocelot.Utils.Html ((<&>))
```

Comment:
![yes-gif](https://user-images.githubusercontent.com/709141/39206748-176eeea6-47b3-11e8-833c-371a75406e35.gif)

I'm all for this PR. The only thing that bothers me is the titlecase `Html` right below the all caps `HTML`

Reply:
I'm not sure that `Html` even adequately describes the module -- it's to do with building blocks and using CSS and adding properties, so it's _kind_ of right, but I'm open to other names too.

Completely fine with changing to `HTML` in the meantime if you want to update that.